### PR TITLE
New website. Loads next meetup from meetup.com

### PR DESCRIPTION
**Pull request by mrpotes**
@sammyd @iamniche @ceejsmith Please can you review the content I've added?

### CODE REVIEW

**Comment by mrpotes** [comment]
The only file you really need to review is index.html:
https://github.com/bris-tech/bris-tech.github.io/pull/9/files#diff-97

File: index.html [hunk 1]
```diff
@@ -1,43 +1,409 @@
...
+						</div>
+						<div class="clearfix"> </div>
+						<p>
+							In 2015, the <a href="http://2015.briste.ch">first Bristech conference</a> was held at
+							<a href="http://at-bristol.org.uk">At Bristol</a> with over 200 delegates attending 3
+							streams of high quality talks from across a wide spectrum of Software Engineering topics.
+						</p>
+						<p>
+							The Bristech Conferences are <i>by developers, for developers</i> - we're all keen software
+							engineers that firmly believe that there's interesting things to learn from all of our
```
Comment (sammyd) [inline index.html]: there _are_ interesting things?

File: index.html [hunk 2]
```diff
@@ -1,43 +1,409 @@
...
+			<!----- //End-header---->
+			<!---- banner ---->
+			<div class="banner wow fadeIn" data-wow-delay="0.5s">
+				<div class="container">
+					<div class="banner-info text-center">
+						<img src="images/logo.png" alt="Bristech Logo"><br />
+						<span> </span>
+						<p>
+							a group of technically-minded people meeting up once a month for beer,
+							pizza and some (hopefully) informative talks about new and interesting
```
Comment (sammyd) [inline index.html]: I'd take out _(hopefully)_. We're trying to sell it. The talks are informative.

**Comment by sammyd** [comment]
@mrpotes two tiny comments on the copy :point_up: 

Other than that happy to go :+1:

**Comment by mrpotes** [comment]
Thanks Sam - updated.

File: index.html [hunk 3]
```diff
@@ -1,43 +1,409 @@
...
+			<!---- banner ---->
+			<div class="banner wow fadeIn" data-wow-delay="0.5s">
+				<div class="container">
+					<div class="banner-info text-center">
+						<img src="images/logo.png" alt="Bristech Logo"><br />
+						<span> </span>
+						<p>
+							a group of technically-minded people meeting up once a month for beer,
+							pizza and some informative talks about new and interesting
+							things that can be done in the world of computer programming
```
Comment (ceejsmith) [inline index.html]: I'm wondering if we're a bit more than just computer programming, e.g. hardware, ethics, soft skills? Maybe "the world of tech"?

**Comment by ceejsmith** [comment]
Looks good to me. I wonder if we replace "computer programming" with "tech" / "technology", but other than that it looks fine.

File: index.html (hunk 3 above)
Comment (mrpotes) [inline index.html]: @ceejsmith how about "software development" then?

Comment (mrpotes) [inline index.html]: or "computer tech"

Comment (ceejsmith) [inline index.html]: "software development" gets my vote

**Pull request merged**